docs/modules/ROOT/pages/reference/vacuum.adoc

```diff
@@ -0,0 +1,325 @@
+
+An empty list of namespaces in the `ClusterVacuumConfig` indicates that
+all namespaces configured for KubeArchive should be vacuumed.
+
+A list of namespaces in the `ClusterVacuumConfig` indicates that only
+those namespaces shoudle be vacuumed.
+
+A namespace with no resources specified indicates that all resources
+defined in the `ClusterKubeArchiveConfig` and the namespace `KubeArchiveConfig`
+should be archived.
+
+A namespace with specific resources defined indicates that only those
+resources in the namespace should be vacuumed.
+
+The special namespace "___all-namespaces___" indicates that all namespaces
+should be processed. As a namespace, "___all-namespaces___" may also 
+specify a list of resources. If "___all-namespaces___" is specified,
+then all namespaces *not* listed in the `ClusterVacuumConfig` will be 
+vacuumed according to the settings for "___all-namespaces___" and
+all listed namespaces will be vacuumed as configured.
+
+The following `ClusterVacuumConfig` will vacuum all resources
+in all namespaces configured for KubeArchive.
+
+.Vacuum all configured resources for all namespaces
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: ClusterVacuumConfig
+metadata:
+  name: cluster-config
+  namespace: kubearchive
+spec:
+  namespaces: {}
+----
+
+The following shows how to vacuum all resources in the three
+specified namespaces.
+
+.Vacuum all resources in three namespaces
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: ClusterVacuumConfig
+metadata:
+  name: cluster-config
+  namespace: kubearchive
+spec:
+  namespaces:
+    namespace-1: {}
+    namespace-2: {}
+    namespace-3: {}
+----
+
+The following shows how to vacuum all resources in two
+namespaces, but only `Job` resources in a third..
+
+.Vacuum some resources in three namespaces
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: ClusterVacuumConfig
+metadata:
+  name: cluster-config
+  namespace: kubearchive
+spec:
+  namespaces:
+    namespace-1: {}
+    namespace-2: {}
+    namespace-3:
+      resources:
+        - apiVersion: batch/v1
+          kind: Job
+----
+
+The following shows how to vacuum all `Job` resources in
+all namespaces not listed, all resources in namespace "namespace-1",
+and `Pod` resources in namespace "namespace-2".
+
+.Vacuum all namespaces with custom configuration for others
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: ClusterVacuumConfig
+metadata:
+  name: cluster-config
+  namespace: kubearchive
+spec:
+  namespaces:
+    ___all-namespaces___:
+      resources:
+        - apiVersion: batch/v1
+          kind: Job
+    namespace-1: {}
+    namespace-2:
+      resources:
+        - apiVersion: v1
+          kind: Pod
+----
+
+==== Sample Cluster Vacuum Cronjob
+Following is sample Cronjob that can be used to run a cluster vacuum.
+The following fields in the `Cronjob` may need to be modified.
```

Comment:
I think here we should specify why the user would want to create a CronJob of its own, since there is already one configured by default.

Reply:
Good point, but note that the sample job configured is suspended so it won't run.  I'll add that too.